.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: go
 go:
-- 1.7.4
+- 1.8.3
```

Comment:
As this is for all intents and purposes a library project, would it be useful to build and test against multiple versions?
i.e.
```
- 1.8.x
- 1.7.x
```